src/vue-components/wordforms.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+    <div class="alpheios-inflections__forms-cont">
+        <span class="alpheios-inflections__forms-targetword">"{{ targetWord }}"</span>
+        <span v-if="forms && forms.length >0" class="alpheios-inflections__form-parts">
+          <span>(</span>
+          <span class="alpheios-inflections__form-part" v-for="(form, index) in forms">
+            {{ form }}<span v-if="index < forms.length-1">, </span>
+          </span>
+          <span>)</span>
+        </span>
+    </div>
+</template>
+<script>
+  import {Constants} from 'alpheios-data-models'
+
+  export default {
+    name: 'WordForms',
+
+    props: {
+      // This will be an InflectionData object
+      partOfSpeech: {
+        type: String,
+        required: true
+      },
+      targetWord: {
+        type: String,
+        required: true
+      },
+      lexemes: {
+        type: Array,
+        required: true
+      }
+    },
+    computed: {
+      forms: function () {
+      	if (this.lexemes) {
+          return this.defineFormsBySelectedView()
+        }
+      }
+    },
+    methods: {
+      defineFormsBySelectedView: function () {
+        let forms = new Set()
+        for (let lexeme of this.lexemes) {
+          for (let inflection of lexeme.inflections) {
+          	if (inflection['part of speech'].values.includes(this.partOfSpeech)) {
```

Comment:
I think it's better to use `Feature.types.part` instead of `'part of speech'` here

Reply:
agree

Reply:
Done